packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -1310,7 +1313,9 @@ function parseRateLimitError(e: Error): RateLimits | undefined {
  * @param {GCFLogger} logger The logger to log to
  * @param {Error} e The error to log
  */
-function logErrors(logger: GCFLogger, e: Error) {
+export function logErrors(logger: GCFLogger, e: Error) {
+  // Add "@type" bindings so that Cloud Error Reporting will capture these logs.
+  logger.addBindings({'@type': ERROR_REPORTING_TYPE_NAME});
```

Comment:
This is modifying the logger instance and could cause unexpected behavior elsewhere. `logger.child(additionalBindings)` will give you a new logger instance with extra bindings (we may want to skip the recursion on line 1323 if we do that)

Reply:
Now it's using `logger.child`